packages/bezier-react/src/components/AlphaSpinner/Spinner.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import { render } from '~/src/utils/test'
+
+import { SPINNER_TEST_ID, Spinner } from './Spinner'
+
+describe('Spinner >', () => {
+  const renderSpinner = (props?: React.ComponentProps<typeof Spinner>) =>
+    render(<Spinner {...props} />)
+
+  it('should render', () => {
+    const { getByTestId } = renderSpinner()
+    const renderedSpinner = getByTestId(SPINNER_TEST_ID)
+    expect(renderedSpinner).toBeInTheDocument()
+  })
+
+  it('should render as a div by default', () => {
+    const { getByTestId } = renderSpinner()
+    const renderedSpinner = getByTestId(SPINNER_TEST_ID)
+    expect(renderedSpinner.tagName).toBe('SPAN')
+  })
```

Comment:
테스트 이름과 수행하는 로직이 다릅니다.

```suggestion
  it('should render as a span by default', () => {
    const { getByTestId } = renderSpinner()
    const renderedSpinner = getByTestId(SPINNER_TEST_ID)
    expect(renderedSpinner.tagName).toBe('SPAN')
  })
```